src/translations/wip.wizards.json

```diff
@@ -1,96 +1,99 @@
 {
+	"htp_metadata_version": "1.0.0",
+    "htp_metadata_credits": "This translation was made by the HTP (Hispanic Translations Projects) Team. \nTranslators: 4drian",
+
     "itemGroup.wizards.general": "Wizards",
   
-    "item.wizards.arcane_spell_book": "Tome of Arcane",
-    "item.wizards.fire_spell_book": "Tome of Fire",
-    "item.wizards.frost_spell_book": "Tome of Frost",
+    "item.wizards.arcane_spell_book": "Tomo Arcano",
+    "item.wizards.fire_spell_book": "Tomo Ígneo",
+    "item.wizards.frost_spell_book": "Tomo Gélido",
```

Comment:
Preferiblemente seguir con la consistencia de usar Escharcha para Frost

Reply:
Vale, siendo mas literales, utilizare "de Fuego" y "de Escarcha" para "of Fire" y "of Frost".

Para mi gusto queda algo feo con los báculos mejorados "Báculo de Fuego de Rubí" porque usa mucho la preposición "de", pero creo que con el input que me das es el mejor compromiso.